paymentsheet/src/test/java/com/stripe/android/paymentsheet/LinkHandlerTest.kt

```diff
@@ -77,60 +73,6 @@ class LinkHandlerTest {
         assertThat(savedStateHandle.get<PaymentSelection>(SAVE_SELECTION)).isNull()
     }
 
-    @Test
-    fun `Completed result sets processing state to Completed`() = runLinkTest {
-        handler.setupLink(
-            createLinkState(LinkState.LoginState.LoggedIn),
-        )
-        handler.launchLink()
-        assertThat(processingStateTurbine.awaitItem()).isEqualTo(LinkHandler.ProcessingState.Launched)
-        verify(linkLauncher).present(configuration)
-        verifyNoMoreInteractions(linkLauncher)
-        handler.onLinkActivityResult(LinkActivityResult.Completed(mock()))
-        assertThat(processingStateTurbine.awaitItem()).isInstanceOf<
-            LinkHandler.ProcessingState.PaymentMethodCollected
-            >()
-    }
-
-    @Test
-    fun `Back pressed result sets processing state to Cancelled`() = runLinkTest {
-        handler.setupLink(
-            createLinkState(LinkState.LoginState.LoggedIn),
-        )
-        handler.launchLink()
-        assertThat(processingStateTurbine.awaitItem()).isEqualTo(LinkHandler.ProcessingState.Launched)
-        handler.onLinkActivityResult(LinkActivityResult.Canceled(LinkActivityResult.Canceled.Reason.BackPressed))
-        assertThat(processingStateTurbine.awaitItem()).isEqualTo(LinkHandler.ProcessingState.Cancelled)
-    }
-
-    @Test
-    fun `Cancelled result sets processing state to Cancelled`() = runLinkTest {
-        handler.setupLink(
-            createLinkState(LinkState.LoginState.LoggedIn),
-        )
-        handler.launchLink()
-        assertThat(processingStateTurbine.awaitItem()).isEqualTo(LinkHandler.ProcessingState.Launched)
-        handler.onLinkActivityResult(LinkActivityResult.Canceled(LinkActivityResult.Canceled.Reason.LoggedOut))
-        val resultState =
-            processingStateTurbine.awaitItem() as LinkHandler.ProcessingState.CompletedWithPaymentResult
-        assertThat(resultState.result).isEqualTo(PaymentResult.Canceled)
-    }
-
-    @Test
-    fun `exception causes processing state to be CompletedWithPaymentResult`() = runLinkTest {
-        handler.setupLink(
-            createLinkState(LinkState.LoginState.LoggedIn),
-        )
-        handler.launchLink()
-        assertThat(processingStateTurbine.awaitItem()).isEqualTo(LinkHandler.ProcessingState.Launched)
-        handler.onLinkActivityResult(LinkActivityResult.Failed(AssertionError("Expected payment result error.")))
-        val result =
-            processingStateTurbine.awaitItem() as LinkHandler.ProcessingState.CompletedWithPaymentResult
-        val paymentResult = result.result as PaymentResult.Failed
-        assertThat(paymentResult.throwable).hasMessageThat()
-            .isEqualTo("Expected payment result error.")
-    }
-
```

Comment:
No loss of coverage from removing these as they tested processing state in the `LinkHandler`, which is no longer relevant after moving `Link` to confirmation definitions.